samples/rtl/rtl_multi_tasklet.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+"""
+    Two sequential RTL tasklets connected through a memlet.
+"""
+
+import dace
+
+import numpy as np
+
+# add sdfg
+sdfg = dace.SDFG('rtl_multi_tasklet')
+
+# add state
+state = sdfg.add_state()
+
+# add arrays
+sdfg.add_array('A', [1], dtype=dace.int32)
+sdfg.add_array('B', [1], dtype=dace.int32)
+sdfg.add_array('C', [1], dtype=dace.int32)
+
+# enable debugging output
+sdfg.add_constant("DEBUG", 1)
```

Comment:
Isn't this a DaCe configuration variable now?

Reply:
It is in on the project todo list to add this as a `dace.conf` parameter. For the time being, i will add it as a command line parameter with default value=false.